kirega/iReporter
# Ft delete incident #162181736

#### What does this PR do?
It creates the necessary endpoints for deleting a records.
`DELETE /api/v1/incident/1/`
#### Description of Task to be completed?
- Creates the incidents model
- writes the necessary tests
- Creates the DELETE endpoint
#### How should this be manually tested?
Clone the repo.
`cd iReporter`
`pip install -r requirements.txt`
`flask run`
using postman or other appropriate endpoints tester,
post  the payload as:
`{
"userid": 1 #user who created the record
}`
to this endpoints :
` /api/v1/incident/1/`
respose expected is
`{   "message":"Incident record has been deleted",
                            "status": 204,
                            "id":incidentId}`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162181715
